### PR TITLE
chore(ims/image_copy): update resource document and some formatting standards

### PR DESCRIPTION
**Pull request by jinyangyang222**
**What this PR does / why we need it**:

1.  Update resource document and some formatting standards
2. Improve test cases
3. `enterprise_project_id` field support update

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.

```
$ export HW_REGION_NAME=cn-north-4
$ export HW_DEST_REGION=cn-south-1

$ make testacc TEST="./huaweicloud/services/acceptance/ims" TESTARGS="-run TestAccImageCopy_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ims -v -run TestAccImageCopy_basic -timeout 360m -parallel 4
=== RUN   TestAccImageCopy_basic
=== PAUSE TestAccImageCopy_basic
=== CONT  TestAccImageCopy_basic
--- PASS: TestAccImageCopy_basic (651.64s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ims       651.684s

$ make testacc TEST="./huaweicloud/services/acceptance/ims" TESTARGS="-run TestAccImageCopy_cross_region_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ims -v -run TestAccImageCopy_cross_region_basic -timeout 360m -parallel 4
=== RUN   TestAccImageCopy_cross_region_basic
=== PAUSE TestAccImageCopy_cross_region_basic
=== CONT  TestAccImageCopy_cross_region_basic
--- PASS: TestAccImageCopy_cross_region_basic (897.54s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ims       897.574s

$ make testacc TEST="./huaweicloud/services/acceptance/ims" TESTARGS="-run TestAccImageCopy_cross_region_withVaultId_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ims -v -run TestAccImageCopy_cross_region_withVaultId_basic -timeout 360m -parallel 4
=== RUN   TestAccImageCopy_cross_region_withVaultId_basic
=== PAUSE TestAccImageCopy_cross_region_withVaultId_basic
=== CONT  TestAccImageCopy_cross_region_withVaultId_basic
--- PASS: TestAccImageCopy_cross_region_withVaultId_basic (1204.71s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ims       1204.774s

```

* [x] Documentation updated.
* [ ] Schema updated.
* [x] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
image

  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
image

### CODE REVIEW

File: docs/resources/images_image_copy.md [hunk 1]
```diff
@@ -52,31 +53,33 @@ The following arguments are supported:
...
 
 * `target_region` - (Optional, String, ForceNew) Specifies the target region name.
   If specified, it means cross-region replication.
 
 * `kms_key_id` - (Optional, String, ForceNew) Specifies the master key used for encrypting an image.
   Only copying scene within a region is supported. Changing this parameter will create a new resource.
 
-* `enterprise_project_id` - (Optional, String, ForceNew) Specifies the enterprise project id of the image.
-  Only copying scene within a region is supported. Changing this parameter will create a new resource.
+* `enterprise_project_id` - (Optional, String) Specifies the enterprise project id of the image.
```
Comment (deer-hang) [inline docs/resources/images_image_copy.md]:
```suggestion
* `enterprise_project_id` - (Optional, String) Specifies the enterprise project ID of the image.
```

File: huaweicloud/services/acceptance/ims/resource_huaweicloud_images_image_copy_test.go [hunk 2]
```diff
@@ -39,18 +39,21 @@ func getImsImageCopyResourceFunc(cfg *config.Config, state *terraform.ResourceSt
...
-	updateName := acceptance.RandomAccResourceName()
-	rName := "huaweicloud_images_image_copy.test"
+func TestAccImageCopy_basic(t *testing.T) {
+	var (
+		obj             interface{}
+		sourceImageName = acceptance.RandomAccResourceName()
+		name            = sourceImageName + "-copy"
+		updateName      = name + "-update"
+		defaultEpsId    = "0"
+		migrateEpsId    = acceptance.HW_ENTERPRISE_PROJECT_ID_TEST
```
Comment (deer-hang) [inline huaweicloud/services/acceptance/ims/resource_huaweicloud_images_image_copy_test.go:49]: Please pre check environment variable `HW_ENTERPRISE_PROJECT_ID_TEST`.

Comment (jinyangyang222) [reply huaweicloud/services/acceptance/ims/resource_huaweicloud_images_image_copy_test.go:49]: done.

**Pull request merged**